sunscreen_docs/src/from_programs_to_apps/chaining.md

```diff
@@ -0,0 +1,30 @@
+# Chaining
+
+Suppose you want to run one program and its output as the input to another program. Sunscreen refers to this pattern as *chaining*. For example, suppose we have an account balance and periodically need to add incoming transactions. We can write a simple FHE program that returns the new balance after adding a transaction:
```

Comment:
"We can write a simple FHE program that returns the new balance after adding a transaction"

"We can express this with the following FHE program:"